docs/source/algorithms/MonteCarloAbsorption-v1.rst

```diff
@@ -20,7 +20,11 @@ spectra in turn:
    and an outgoing vector to the final detector;
 -  The total attenuation factor for this path is then calculated as the
    product of the factor for each defined material of the
-   sample/container that the track passes through.
+   sample/container that the track passes through.  Any process that 
+   removes neutrons from the beam prior to the scattering point looks
+   like absorption.  Therefore both the absorption cross section are 
```

Comment:
Oops yes that is what I meant and I corrected the code to reflect the proper comment.